# Define the goal
string database             # database file name
uint16 model_id             # ID of model according to which movement is executed
float64 action_variable     # describes the movement relative to the start pose 
                            # (e.g. opening angle in rad for a rotational articulation 
                            # or distance in m for a prismatic one) 
duration duration           # duration in secs until movement should be finished
---
# Define the result
bool success
string error_message
---
# Define a feedback message
string message
